kong/plugins/proxy-cache/schema.lua

```diff
@@ -84,6 +84,16 @@ return {
             type = "array",
             elements = { type = "string" },
           }},
+          { response_headers = {
+            type = "record",
```

Comment:
Why is an array of header keys not sufficient?

Reply:
To explicit the expected headers in this field.

To keep previous behavior, not setting a value for an header should be seen as true.
An empty list => all headers should be in the response.

Reply:
This isn't ideal but I can live with it. Resolving thread.